src/style/themes/classic/classic-theme.config.js

```diff
@@ -9,7 +9,14 @@ export default (palette) => {
         base: palette.productGreen,
         primary: palette.productGreenShade(21),
         secondary: palette.productGreenShade(41),
-        tertiary: palette.productGreenShade(61)
+        tertiary: palette.productGreenShade(61),
+
+        iconBackground: '#e6ebed',
```

Comment:
Is there a reason for declaring some colour hexes here and some in the palette? Might be better to be consistent one way or the other.

Reply:
I've just moved the new one I added to the palette to a hard-coded definition in here as it wasn't being used elsewhere. The other ones I've used from the palette were already there, and the rest I hard-coded in here for now as they weren't being re-used.

Reply:
I can move them all to one place or the other here if that would be preferred? Just seemed like the palette was meant only to be used for things which were being re-used across other styles.

Will take your lead on it though 👍 do you have a preference one way or the other?